frontend/views/components/modal/Modal.vue

```diff
@@ -128,16 +129,21 @@ export default ({
         if (this.lastFocus) this.lastFocus.focus()
       }
       if (this.replacement) {
-        this.openModal(this.replacement)
+        this.openModal(this.replacement, this.replacementQueries[this.replacement])
+
         this.replacement = null
+        delete this.replacementQueries[this.replacement]
```

Comment:
Seems weird here. We should delete `this.replacement` first and then set it `null`, shouldn't we?

Reply:
@Silver-IT yeah right. good catch :)